tests/system/test-ingest.js

```diff
@@ -476,16 +478,17 @@ test('Ingested item is published to post-ingest SNS topic', async (t) => {
 })
 
 test('Ingest item failure is published to post-ingest SNS topic', async (t) => {
-  const collection = await ingestCollectionAndPurgePostIngestQueue(t)
+  await ingestCollectionAndPurgePostIngestQueue(t)
```

Comment:
this previously failed because Item was missing a `links` field, so I had to make it fail for another reason, e.g., collection doesn't exist